### DISCUSSION TITLE
Classification demo

### DISCUSSION THREAD

**Discussion by SitSaw17** [bitcraze/discussions#113] [Q&A]
Hi
I was wondering what the confidence value is in the classification example. Usually, the confidence score is between 0 and 1, but the confidence value in the example is way beyond that.

**Comment by knmcguire** [comment]
Good question! 

I believe this is because the network has been quantized to either 16 or 8 bit values in order to properly fit on the gap8 chip, so then it makes sense that the output is an short integer type as well. Then 0 is just 0 and 1.0 is equivalent to 32767 (the maximum value of a short signed int which is[ the variable type of output](https://github.com/bitcraze/aideck-gap8-examples/blob/204aaff2c183e26bf109f7c6201dd2f045916f46/examples/ai/classification/classification.c#L54)).

@gemenerik please correct me when I'm wrong :)
